<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controlador</class>
 <widget class="QWidget" name="Controlador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbAvanzar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Avanzar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbIzquierda">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Izquierda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDerecha">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Derecha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRetroceder">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Retroceder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
